src/app/shared/navbar/searchbar/searchbar.scss

```diff
@@ -0,0 +1,62 @@
+$input-bg: #85D9E2;
+
+@mixin color-placeholder() {
+  -webkit-font-smoothing: antialiased;
+  color: white;
+}
+
+:host {
+  position: relative;
+  flex: 2;
+
+  * {
+    box-sizing: border-box;
+  }
+
+  &.expanded .search-input-container {
```

Comment:
All of the css should start with a `docs-` prefix
(mainly so that people looking at the docs site in dev tools can differentiate between docs-site-specific css and the underlying library css)